### ISSUE TITLE
Training code release

### ISSUE THREAD

**Issue by hdjsjyl** [showlab/DIM#4]
Hi Author,
    Thanks for excellent work. 
    Is there any plan to release training code? 
    Thanks

**Comment by stdKonjac** [comment]
Hi, thank you for your interest in our work. Currently, we do not have plans to release the training code because it is built on internal packages & framework. Other sources, e.g., DIM models and DIM-Edit, will be released possibly within two weeks, depending on the review process :) BTW, from my own experience, you may simply use a HuggingFace trainer to implement training.

**Comment by stdKonjac** [comment]
> hi [@stdKonjac](https://github.com/stdKonjac) , really appreciate your reply. That is my plan. I just want to make sure the details are correct. Maybe I will ask for help from you to review my training code. Thanks

Hi, we have released the DIM-Edit dataset and DIM-4.6B-T2I/Edit models. To facilitate reproducibility, we release DIM-4.6B-Edit-Stage1, which is trained solely on the UltraEdit dataset. By fine-tuning this checkpoint on our proposed DIM-Edit dataset, you should obtain DIM-4.6B-Edit (so that you do not need many GPUs to train on the 4M UltraEdit data).
